src/app/components/docsUtil/rules/TaggedCaption.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { ListItemTag } from '@brightlayer-ui/react-components';
+import { PaletteColor } from '@material-ui/core/styles/createPalette';
+import { useTheme, Tooltip } from '@material-ui/core';
+import { orange } from '@brightlayer-ui/colors';
+import RULES from './rules';
+
+export type RuleType = `DON'T` | 'AVOID' | 'CAUTION' | 'DO';
```

Comment:
For string literal types, I recommend using lowercase/snakecase for the values to make them easier to work with. Having the `'` in a string literal will be annoying because you either have to always use a template literal or escape the character.
```suggestion
export type RuleType = `do-not` | 'avoid' | 'caution' | 'do';
```

Reply:
These texts are used directly in list item tags... I don't feel like adding another layer of logic